src/components/Observations/map/TooltipContents.tsx

```diff
@@ -1,48 +1,92 @@
 import strings from 'src/strings';
-import { ObservationState } from 'src/types/Observations';
+import { ObservationState, ObservationMonitoringPlotResultsPayload } from 'src/types/Observations';
+import ReplaceObservationPlotModal from 'src/components/Observations/replacePlot/ReplaceObservationPlotModal';
+
+const useStyles = makeStyles(() => ({
+  button: {
+    marginLeft: 'auto',
+    '&:focus': {
+      outline: 'none',
+    },
+  },
+}));
 
 type TooltipContentsProps = {
-  title: string;
+  monitoringPlot: ObservationMonitoringPlotResultsPayload;
+  observationId?: number;
   observationState?: ObservationState;
-  numPlants?: number;
-  numSpecies?: number;
-  plantingDensity?: number;
-  percentMortality?: number;
+  title: string;
 };
 
 export default function TooltipContents(props: TooltipContentsProps): JSX.Element {
-  const { title, observationState, numPlants, numSpecies, plantingDensity, percentMortality } = props;
+  const { monitoringPlot, observationId, observationState, title } = props;
   const theme = useTheme();
+  const classes = useStyles();
+  const [showReplacePlotModal, setShowReplacePlotModal] = useState<boolean>(false);
 
   const observationInProgress = observationState === 'InProgress';
   const observationOverdue = observationState === 'Overdue';
 
+  const numPlants = monitoringPlot?.totalPlants;
+  const numSpecies = monitoringPlot?.totalSpecies;
+  const plantingDensity = monitoringPlot?.plantingDensity;
+  const percentMortality = monitoringPlot?.mortalityRate;
+
   return (
-    <Box display='flex' flexDirection='column' padding={theme.spacing(1)}>
-      <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(2)}>
-        {title}
-      </Typography>
-      {observationInProgress ? (
-        <Typography>{strings.OBSERVATION_IN_PROGRESS}</Typography>
-      ) : observationOverdue ? (
-        <Typography>{strings.OBSERVATION_OVERDUE}</Typography>
-      ) : (
-        <>
-          <Typography fontSize='16px' fontWeight={400}>
-            {`${strings.PLANTS}: ${numPlants ?? strings.UNKNOWN}`}
-          </Typography>
-          <Typography fontSize='16px' fontWeight={400}>
-            {`${strings.SPECIES}: ${numSpecies ?? strings.UNKNOWN}`}
-          </Typography>
-          <Typography fontSize='16px' fontWeight={400}>
-            {`${strings.PLANTING_DENSITY}: ${plantingDensity ?? strings.UNKNOWN} ${strings.PLANTS_PER_HECTARE}`}
-          </Typography>
-          <Typography fontSize='16px' fontWeight={400}>
-            {`${strings.MORTALITY_RATE}: ${percentMortality ?? strings.UNKNOWN}%`}
-          </Typography>
-        </>
+    <>
+      {showReplacePlotModal && observationId && monitoringPlot && (
+        <Portal>
+          <ReplaceObservationPlotModal
+            onClose={() => setShowReplacePlotModal(false)}
+            observationId={observationId}
+            monitoringPlot={monitoringPlot}
+          />
+        </Portal>
```

Comment:
Using a react portal to render this modal outside the current tooltip modal's DOM hierarchy, so it has the entire screen estate.. otherwise it is super squished inside the tiny tooltip modal